### PR TITLE
Added web3uikit build setup issues to contribute md

### PR DESCRIPTION
**Pull request by nikita12c**
---
name: 'Pull request'
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

While contributing to this project initially , few issues are faced by new contributors. Those are some version related issues or build time issues while doing initial setup of web3uikit project.

Related issue: #817

### Solution Description

Added some commands which will help new contributors to solve the version issue or build issues to some extent.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -37,13 +37,26 @@ You want to get started? Follow these 3 simple steps
...
 
 &nbsp;
 
+# Getting error while doing above project setup 
+
+-   You can download nvm to switch between the node versions. Try to be at node version 16. [`nvm install 16` then `nvm use 16` check
+    it using `node -v`, make sure it is node version 16].
+-   Facing access denied issue from nvm then open the terminal as run by administrator then try running above commands.
+-   Now follow three simple steps again -
+     1) Run `pnpm install`
```
Comment (AbhinavMV) [inline CONTRIBUTE.md:54]: This can be removed since it is already mentioned above :)

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,140 @@
+Arguments:
```
Comment (AbhinavMV) [inline yarn-error.log:1]: Can you remove this file please?

Comment (nikita12c) [reply yarn-error.log:1]: Oh yes  ! I remove it

File: CONTRIBUTE.md (hunk 1 above)
Comment (nikita12c) [reply CONTRIBUTE.md:54]: Should I remove Line no. 51 to 54 , or the 3 simple steps repeated from Line 53 to 56 ?

Comment (nikita12c) [reply CONTRIBUTE.md:54]: Okay

**Review by nikita12c — COMMENTED** [review]
Done with the changes

File: yarn-error.log (hunk 2 above)
Comment (nikita12c) [reply yarn-error.log]: Done

File: CONTRIBUTE.md (hunk 1 above)
Comment (nikita12c) [reply CONTRIBUTE.md]: Done

**Comment by nikita12c** [comment]
Removed line 53 in contribute md file

**Review by nikita12c — COMMENTED** [review]
Removed line 53 in contribute.md

**Review by AbhinavMV — APPROVED** [review]
LGTM 🚀 

TY for your contribution 😄

**Pull request merged**